geode-core/src/distributedTest/java/org/apache/geode/internal/cache/execute/NestedFunctionExecutionDistributedTest.java

```diff
@@ -111,6 +116,7 @@ private int createServerCache() throws IOException {
 
   private void createClientCache(int port, int functionTimeout) {
     System.setProperty(GEMFIRE_PREFIX + "CLIENT_FUNCTION_TIMEOUT", String.valueOf(functionTimeout));
+    System.setProperty(MAX_FE_THREADS_PROPERTY, TEST_MAX_FE_THREADS);
```

Comment:
Its not necessary to set this value in the client. The Function Execution thread pool is not created there.

Reply:
Currently the client does `MAX_FE_THREADS * 2` executions of `ParentFunction`. If I don't set `MAX_FE_THREADS` in the client, I get a swarm of failures with that many functions. Should I directly reduce the number of functions it executes, rather than computing from `MAX_FE_THREADS`?

Reply:
Sorry, you're right. You should set it.